toolchest_client/tools/tool.py

```diff
@@ -400,6 +401,7 @@ def run(self):
                 "output_name": f"{index}_{self.output_name}" if should_run_in_parallel else self.output_name,
                 "input_files": input_files,
                 "input_prefix_mapping": self.input_prefix_mapping,
+                "input_prefix_order": self.input_prefix_order,
```

Comment:
I think it makes sense to add the input prefix to `input_prefix_mapping`. For an example, see the worker node PR

Reply:
Gotcha, I'll do a quick refactor